File: Mirrah Day-03 Sunburnt/solution.cpp
#include<iostream>
#include<vector>
#include<sstream>
using namespace std;

int main(){
    int N,n,count,maximum;
    cin>>N;
    cin.ignore();
    vector<int> H;
    string input;
    getline(cin,input);
    istringstream iss(input);
    while (iss>>n){
        H.push_back(n);
    }
    if (H.empty()) count=0;

    else{
        count=1;
        maximum=H[0];
        for (int i=1;i<H.size();i++){
            if (H[i]>=maximum){
                maximum=H[i];
                count++;
            }
        }
    }
    cout<<count;
    return 0;
}

File: Mirrah Day-03 Sunburnt/solution.py
N=int(input())
H=[int(b) for (a,b) in enumerate(input().split()) if b.isdigit() and a<N]
if len(H)==0:
    count=0
else:
    count=1
    maximum=H[0]
    for i in range(1,len(H)):
        if H[i]>=maximum:
            maximum=H[i]
            count+=1
print(count)